.github/codecov.yaml

```
@@ -0,0 +1,59 @@
+# Find more at https://docs.codecov.com/docs/codecovyml-reference
+codecov:
+  bot: "Codecov Bot"
+  max_report_age: 12
+  require_ci_to_pass: yes
+  notify:
+    after_n_builds: 1
+    wait_for_ci: yes
+
+# Layout of the PR comment produced by Codecov bot
+comment:
+  layout: "header, diff, flags, components, files"
+
+ignore:
+  - api/external/** #  ignoring external vendor code
+
+flag_management:
+  individual_flags:
+    - name: unit
+      paths:
+        - pkg/**
+        - api/**
+      carryforward: true
+    - name: integration
+      paths:
+        - controllers/**
+      carryforward: true
+
+component_management:
+  individual_components:
+    - component_id: api-v1beta1
+      name: api/v1beta1
```

```suggestion
      name: api/v1beta1 (u)
```

nit: thoughts on adding `(u)` like the other componants for consistency ?

---

Thanks! I'm adding that!